### DISCUSSION TITLE
What about the Q1 Pro?

### DISCUSSION THREAD

**Discussion by Wuehli** [leadustin/QIDI-up2date-german#5] [General]
Hi,

i would like to ask if this would/could also work on the Q1 Pro?

regards
W

**Comment by leadustin** [comment]
It could also work with the Q1. However, I don't have the printer and therefore can't test extensively whether everything works. As I don't want to ruin other people's printers, I can't publish a guide at the moment.

Es könnte auch mit dem Q1 funktionieren. Allerdings habe ich den Drucker nicht und kann daher nicht ausgiebig testen ob alles funktioniert. Da ich nicht andere Leute ihren Drucker schrotten möchte, kann ich zum derzeitigen Zeitpunkt keinen Guide veröffentlichen.

**Comment by Wuehli** [comment]
Hi,

Unfortunately, I still have quite an offset / first layer problem with my Q1 at the moment. As soon as I am through with the support I would be willing to sacrifice my printer as a testing ground. However, it is my first printer with Klipper, so I would need some guidance here. 
(Gern auch in deutsch 😃 )
regards
W

**Reply by leadustin** [reply]
Hi, 
das Offset musst du immer am Display einstellen, nicht per Webinterface Fluidd. Da gibt es häufig Probleme. Allerdings wird dir ein Umbau die Probleme mit dem Offset nicht unbedingt lösen. Häufig ist es der faslche Abstand der Düse zum Druckbett. Aber wenn du aus der Garantie heraus bist, kann ich gern mit dir, sofern es dann nicht bereits einen Guide gibt, den Umbau zusammen durchführen.

**Reply by Wuehli** [reply]
Problem gelöst, eMMC Reader ist bestellt, sobald der da ist kann es losgehen.
